frontend/src/FullPageRoutes/FAQ.tsx

```diff
@@ -0,0 +1,169 @@
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    One (1) video messages be per day can be sent.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>How long can the video be?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The video can be between fifteen seconds to 10 minutes long.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+            </Grid>
+
+            <Grid item>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>What are the privacy settings?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    The privacy settings are public, private and hidden. Public
+                    allows for everyone to see your information. Private allows
+                    for just my penpals to see your information. Hidden allows
+                    for no one to see your information.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>Can I decline a penpal?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>Yes.</Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>
+                    Is there a maximum of penpals I can accept in a day?
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>No.</Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>Can you edit videos?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>Yes.</Typography>
```

Comment:
"Yes, there are tools available to edit videos in the video recording part of the site."
Later we need to list what those tools are and explain a bit more.